aerospike_enterprise/metadata.csv

```diff
@@ -0,0 +1,651 @@
+aerospike.namespace.from_proxy_lang_error,gauge,,error,,The number of language (Lua) errors for udf transactions proxied from another node.,0,aerospike,,,
+aerospike.namespace.from_proxy_lang_read_success,gauge,,transaction,,The number of successful udf read transactions proxied from another node.,0,aerospike,,,
+aerospike.namespace.from_proxy_lang_write_success,gauge,,transaction,,The number of successful udf write transactions proxied from another node.,0,aerospike,,,
+aerospike.namespace.from_proxy_read_error,gauge,,transaction,,The number of errors for read transactions proxied from another node.,0,aerospike,,,
+aerospike.namespace.from_proxy_read_filtered_out,gauge,,transaction,,The number of read transactions proxied from another node that did not happen because the record was filtered out via a predicate expression.,0,aerospike,,,
+aerospike.namespace.from_proxy_read_not_found,gauge,,transaction,,The number of read transactions proxied from another node that resulted in not found.,0,aerospike,,,
+aerospike.namespace.from_proxy_read_success,gauge,,transaction,,The number of successful read transactions proxied from another node.,0,aerospike,,,
+aerospike.namespace.from_proxy_read_timeout,gauge,,transaction,,The number of timeouts for read transactions proxied from another node.,0,aerospike,,,
+aerospike.namespace.from_proxy_tsvc_error,gauge,,transaction,,"The number of transactions proxied from another node that failed in the transaction service, before attempting to handle the transaction.",0,aerospike,,,
+aerospike.namespace.from_proxy_tsvc_timeout,gauge,,transaction,,"The number of transactions proxied from another node that timed out while in the transaction service, before attempting to handle the transaction.",0,aerospike,,,
+aerospike.namespace.from_proxy_udf_complete,gauge,,transaction,,The number of successful udf transactions proxied from another node.,0,aerospike,,,
+aerospike.namespace.from_proxy_udf_error,gauge,,transaction,,The number of errors for udf transactions proxied from another node.,0,aerospike,,,
+aerospike.namespace.from_proxy_udf_filtered_out,gauge,,transaction,,The number of udf transactions proxied from another node that did not happen because the record was filtered out via a predicate expression.,0,aerospike,,,
+aerospike.namespace.from_proxy_udf_timeout,gauge,,transaction,,The number of timeouts for udf transactions proxied from another node.,0,aerospike,,,
+aerospike.namespace.from_proxy_write_error,gauge,,transaction,,The number of errors for write transactions proxied from another node. This includes xdr_from_proxy_write_error.,0,aerospike,,,
+aerospike.namespace.from_proxy_write_filtered_out,gauge,,transaction,,The number of write transactions proxied from another node that did not happen because the record was filtered out via a predicate expression.,0,aerospike,,,
```

Comment:
```suggestion
aerospike.namespace.from_proxy_write_filtered_out,gauge,,transaction,,The number of write transactions proxied from another node that did not happen because the record was filtered out using a predicate expression.,0,aerospike,,,
```